src/main/java/seedu/address/logic/commands/viewcommands/ViewPersonCommand.java

```diff
@@ -17,6 +17,7 @@
  */
 public class ViewPersonCommand extends ViewCommand {
     public static final String MESSAGE_SUCCESS = "Person with index %d displayed!";
+    public static final String DISPLAY_MESSAGE_SUCCESS = "You are now viewing the person: ";
```

Comment:
Unfortunately due to the feature freeze, I don't think we're allowed to change the UI output :(